public/static/docs/api-reference/get_url.md

```diff
@@ -4,34 +4,43 @@ Returns the URL to the storage location of a data file or directory tracked in a
 <abbr>DVC project</abbr>.
 
 ```py
-dvc.api.get_url( path: str, repo: str = None,
-    rev: str = None, remote: str = None ) -> str
+def get_url(path: str,
+            repo: str = None,
+            rev: str = None,
+            remote: str = None) -> str
 ```
 
-**Usage:**
+#### Usage:
 
 ```py
 import dvc.api
 
-resource_url = dvc.api.get_url(
+dvc.api.get_url(
     'get-started/data.xml',
     repo='https://github.com/iterative/dataset-registry')
+
+# https://remote.dvc.org/dataset-registry/a3/04afb96060aad90176268345e10355
```

Comment:
Agee about removing the var name but its not very obvious what the comment below is (and probably creates a horizontal scroll). Not sure it's needed for basic usage.

Reply:
I think it's intuitively clear ... we can even bring var name back in the comment ... we need to show something than nothing

we can just add 

`# returns 'http://....'

Reply:
I don't think the basic usage block needs to show the variable value. That's what examples are for I think.

But if you want to keep this I would not reintroduce var name, maybe just make the comment shorter? `# https://remote.dvc.org/dataset-registry/a3/04a...` (maybe in the example BTW, so that no scrolls are shown) but not a huge deal about the scrolls for these cases.

Reply:
I think it makes this example significantly better if we show the value, I would even say that this the most important part  - you see right away what it does

I would keep at as simple as possible, would add `returns` at most

Reply:
OK. Actually I would reintroduce the var name now that I think about it. You wouldn't call a string returning function without assigning it to something. So my full suggestion is:

```py
resource_url = dvc.api.get_url(
    'get-started/data.xml',
    repo='https://github.com/iterative/dataset-registry')
    
# resource_url = https://remote.dvc.org/dataset-registry/a3/04a...
```

Reply:
Or wrap in `print()`

Reply:
don't use `=` though, write it in a plain language - resource_url is now 'http:///'

Reply:
otherwise it looks like a commented code